rpcs3/Emu/RSX/gcm_enums.h

```diff
@@ -992,8 +992,35 @@ enum
 
 enum Method
 {
+	/*
```

Comment:
The best is to keep original constants unchanged, and not try to mimic their style (CELL_*) in newly defined constants. This helps to avoid possible further confusion.